tests/python/dnnl/subgraphs/test_conv_subgraph.py

```diff
@@ -72,6 +72,26 @@ def forward(self, x):
   net = Conv()
   check_fusion(net, data_shape, attr)
 
+@mx.util.use_np
+@pytest.mark.parametrize('data_shape', DATA_SHAPE)
+@pytest.mark.parametrize('use_bias', [True, False])
+def test_conv_transpose_conv(use_bias, data_shape):
+
+  class Conv_Transpose_Conv(nn.HybridBlock):
+    def __init__(self, **kwargs):
+        super(Conv_Transpose_Conv, self).__init__(**kwargs)
+        self.conv0 = nn.Conv2D(channels=64, kernel_size=(3, 3), strides=1, use_bias=use_bias)
+        self.conv1 = nn.Conv2D(channels=32, kernel_size=(5, 5), strides=1, use_bias=use_bias)
+
+    def forward(self, x):
+      out = self.conv0(x)
+      out = mx.np.transpose(out, axes=[0,1,3,2])
+      out = self.conv1(out)
+      return out
+
+  attr = {'conv': []}
+  net = Conv_Transpose_Conv()
+  check_fusion(net, data_shape, attr)
```

Comment:
I think in this case check_quantize call should be used - probably you can check if transpose was quantized by adding it's name to attr dict